bin/build_extras.sh

```diff
@@ -93,7 +93,7 @@ if [ "$1" != "nocmake" ] && [ "$1" != "install" ] ; then
   if [ $AOMP_STANDALONE_BUILD == 1 ] ; then
     MYCMAKEOPTS="-DLLVM_DIR=$LLVM_DIR -DCMAKE_BUILD_TYPE=$BUILDTYPE -DCMAKE_INSTALL_PREFIX=$INSTALL_EXTRAS -DAOMP_VERSION_STRING=$AOMP_VERSION_STRING -DCMAKE_PREFIX_PATH=$BUILD_DIR/build/libdevice"
   else
-    MYCMAKEOPTS="-DLLVM_DIR=$LLVM_DIR -DCMAKE_BUILD_TYPE=$BUILDTYPE -DCMAKE_INSTALL_PREFIX=$INSTALL_EXTRAS -DDEVICELIBS_ROOT=$DEVICELIBS_ROOT -DAOMP_VERSION_STRING=$AOMP_VERSION_STRING"
+    MYCMAKEOPTS="-DLLVM_DIR=$LLVM_DIR -DCMAKE_BUILD_TYPE=$BUILDTYPE -DCMAKE_INSTALL_PREFIX=$INSTALL_EXTRAS -DDEVICELIBS_ROOT=$DEVICELIBS_ROOT -DAOMP_VERSION_STRING=$AOMP_VERSION_STRING -DENABLE_DEVEL_PACKAGE=ON"
```

Comment:
There needs to be some logic here so the options are not always on.
```
if [ "$AOMP_STANDALONE_BUILD" == 0 ]; then
  MYCMAKEOPTS="$MYCMAKEOPTS -DENABLE_DEVEL_PACKAGE=ON"
fi
```

Reply:
Edit: build_extras.sh looks fine in this regard, but the others need changed.

Reply:
define of devel and runtime package, moved to the condition suggested